library/src/main/scala/giter8/GitRepository.scala

```diff
@@ -47,7 +47,7 @@ object GitRepository {
     val NativeUrl: Regex = "^(git[@|://].*)$".r
     val HttpsUrl: Regex  = "^(https://.*)$".r
     val HttpUrl: Regex   = "^(http://.*)$".r
-    val SshUrl: Regex    = "^(ssh://.*)$".r
+    val SshUrl: Regex    = "^ssh://(.*)$".r
```

Comment:
Doing some investigation, seems that the version of jgit is different, we used this one on the PR you mention:
https://github.com/njlg/giter8/blob/cf463d3e50a7a4b40a8fb3ba189091b689bedbc0/project/Dependencies.scala#L16

But now we use this one:
https://github.com/foundweekends/giter8/blob/master/project/Dependencies.scala#L16

Probably the bug was introduced when we changed the jgit dependency.